cmd/poller/plugin/README.md

```diff
@@ -111,6 +113,68 @@ The plugin tries to intelligently aggregate metrics based on a few rules:
 - **Weighted Average** - applied if metric has property `average` and suffix `_latency` and if there is a matching `_ops` metric. (This is currently only matching to ZapiPerf metrics, which use the Property field of metrics.)
 - **Ignore** - metrics created by some plugins, such as value_to_num by LabelAgent
 
+# Max
+
+Max creates a new collection of metrics (Matrix) by calculating max of metric values from an existing Matrix for a given label. For example, if the collected metrics are for volumes, you can create max at nodes or svms level.
+
+### Max Rule syntax
+
+simplest case:
+
+```yaml
+plugins:
+  Max:
+    - LABEL
+# will create a new Matrix based consisting of max values on target label LABEL
+```
+
+If you want to specify which labels should be included in the new instances, you can add those space-seperated after `LABEL`:
+
+```yaml
+    - LABEL LABEL1,LABEL2
+# same, but LABEL1 and LABEL2 will be copied into the new instances
+# (default is to only copy LABEL and any global labels (such as cluster and datacenter)
+```
+
+Or include all labels:
+
+```yaml
+    - LABEL ...
+# copy all labels of the original instance
+```
+By default, metrics will be prefixed with `LABEL`. For example if the object of the original Matrix is `volume` (meaning metrics are prefixed with `volume_`) and `LABEL` is `aggr`, then the metric `volume_read_ops` will become `aggr_volume_read_ops`, etc. You can override this by providing the `<>OBJ` using the following syntax:
```

Comment:
By default, metrics will be prefixed with `LABEL`. For example if the object of the original Matrix is `volume` (meaning metrics are prefixed with `volume_`) and `LABEL` is `aggr`, then the metric `volume_read_ops` will become `aggr_volume_read_ops`. You can override this using the `<>OBJ` pattern shown below: